### PR TITLE
Changes for cura connect integration

### PR DESCRIPTION
**Pull request by woosie10**

### CODE REVIEW

File: src/components/slide_out_container.tsx [hunk 1]
```diff
@@ -50,6 +48,10 @@ export class SlideOutContainer extends React.Component<SlideOutContainerProps, S
     </div>
 
   }
-}
+
+SlideOutContainer.defaultProps = {
+  showHeaderPanelArrow: true,
+  headerTextAlignment: 'left'
+};
```
Comment (ChrisTerBeke) [inline src/components/slide_out_container.tsx]: missing SlideOutContainer.displayName = 'SlideOutContainer'

Comment (ChrisTerBeke) [reply src/components/slide_out_container.tsx]: Also for many other components I see (this makes the snapshots know what the names of components are).

File: src/components/slide_in_panel.tsx [hunk 2]
```diff
@@ -24,6 +24,8 @@ export interface SlideInPanelProps {
 
 export class SlideInPanel extends React.Component<SlideInPanelProps, {}> {
 
+  public static displayName = 'SlideInPanel';
```
Comment (ChrisTerBeke) [inline src/components/slide_in_panel.tsx]: For classes it's not needed as they are already named by their class.

Comment (woosie10) [reply src/components/slide_in_panel.tsx]: Ahhhhhh, I'll remove them

**Pull request merged**